pkg/db/interface.go

```diff
@@ -44,3 +48,15 @@ type DataRows interface {
 	Scan(dest ...interface{}) error
 	Next() bool
 }
+
+func newID() string {
```

Comment:
Why not using uuid package instead? We are already using it in other parts of the project

Reply:
Changed to UUID